splicemachine/ml/.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "python.pythonPath": "/Users/nireksharma/anaconda3/envs/py27/bin/python"
```

Comment:
This is a local path to a config file. I would recommend putting `.vscode/` in the `.gitignore`.